Gemfile

```diff
@@ -4,13 +4,11 @@ source 'http://rubygems.org'
 gem "spree_auth_devise", github: "spree/spree_auth_devise", branch: '1-3-stable'
 
 # we use line_item_description_text which is in newer sprees
-gem 'spree', github: 'freerunningtech/spree', branch: '1-3-stableish'
+gem 'spree', github: 'degica/spree', branch: '1-3-rebased-clean'
 
 group :development, :test do
-  gem 'fuubar'
```

Comment:
@shioyama Alright, I made the first 3 commits that fix dependencies only into a separate PR #2.